locations/darkworld.json

```diff
@@ -493,14 +493,14 @@
             "@Light Death Mountain Left Bottom, pearl",
             // "@Light Death Mountain Left Bottom, $checkGlitches|2", //leave dm without pearl
             "inverted, flute_active",
-            "inverted, $darkRooms, glove"
+            "inverted, [$darkRooms], glove"
         ],
         "children": [
             {
                 "name": "Spike Cave",
                 "access_rules": [
-                    // "hammer, glove",
-                    "$health|14, hammer, glove"
+                    "bottle, invincibility, hammer, glove",
+                    "half_magic, invincibility, hammer, glove"

```

Comment:
I can tell you that this function does not work properly. First, heart pieces are not being counted for total health, at all. I can click that heart full dozens of times and this check will never go on. Additionally, Byrna or Cape + Half/Quarter Magic should toggle the check based on the AP logic, but they don't unless you toggle enough boss hearts (in this case, 5). Also, 14 health on its own does not put this check in AP's logic. If you have 20 heart containers but no invincibility items, this check is not in logic.

Reply:
This was the case before the 0.0.6 release. But i changed the function there and it works. If you toggle 11 hearthconainer so that you have in total 14 health it will toggle green. I will look at the function if i missed something with a commit but it should work.

Reply:
result: assumption of 14 hearts being considered was false. base assumption was wrong and commit is correct